requirements.txt

```diff
@@ -1,11 +1,11 @@
 cycler==0.10.0
 h5py==2.10.0
 kiwisolver==1.2.0
-matplotlib==3.1.2
+matplotlib==3.3.2
```

Comment:
Good question. Matplotlib was raising all kinds of warnings in the notebook, so I just bumped to the recent version. I think I changed pandas because it was the simplest way to get the `atol` parameter in some tests. Given #197 I don't have any hesitation about matplotlib, which is nice!